.vib/gitlab-runner/goss/vars.yaml

```diff
@@ -0,0 +1,11 @@
+binaries:
+  - gitlab-runner
+files:
+  - paths:
+      - /opt/bitnami/gitlab-runner/bin/gitlab-runner
+directories:
+  - mode: "0755"
+    paths:
+      - /opt/bitnami/gitlab-runner
+      - /opt/bitnami/gitlab-runner/bin
```

Comment:
I think there is no explicit logic setting the mode for these paths, am I right?